### DISCUSSION TITLE
How to use GitHub actions rerun a stage

### DISCUSSION THREAD

**Discussion by ukreddy-erwin** [community/community#36809] [Actions]
When a particular stage/job is failed in GitHub actions workflow, we can use the 'rerun failed jobs' option.

But let's say I have 3 jobs one after other i a workflow.

How to run only the last job again later manually even it is successful at present, I can see that option only if the previous job failed.

The last job is mainly for packaging and I am using external runners.

Regards,
Uday

**Comment by ossanna16** [comment]
Thank you for asking a good question @ukreddy-erwin, I'm glad you found our community 🙂
